modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -354,6 +395,69 @@ struct IEUnit {
     }
 };
 
+bool IEUnit::InputFrameDesc::is_applicable(const cv::GMetaArg &mm) {
```

Comment:
Does `preproc` support only `NV12`?

Reply:
`preproc` supports NV12 it is true, but  this part is for other purpose: just collect information in wide way and store it for further preproc appliance. in other words `InputFrameDesc` just describes input frame

And because NV12 is a current temprary preproc engine limitation - it must be implemented in preproc engine `is_applicable` by itself but not here.

P.S. in current implementation limitation on NV12 in PPEngine doens't require any kind of sanitizing check because MediaAdapter in VPP doesn't allow to construct any other kind of MediaFrame formats in source by itself